### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Azure Sign In_ Color Contrast: Color contrast ratio for “Sign In” button is less than 4.5:1.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3319]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.4.3; #Color Contrast; #Java tooling for IntelliJ/Eclipse; #C+E-H

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19
OS Details: 
OS: Windows (Version 1903, Build: 18362.175)
**MAS Violated**: MAS1.4.3

**NOTE:** Issue is observed only in "Dracula" mode.

**Repro Steps:** 
1) Open IntelliJ IDEA application
2) Press 'Ctrl+ Alt+ S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter.
6) Restart IntelliJ IDEA application and enable “Dracula” theme.
7)Tab till “Azure Sign In” button and press enter.
8)Check Color Contrast ratio for “Sign In” buttons.

**Actual Result:**
Color contrast ratio for “Sign In” buttons is less than 4.5:1.

**Expected Result:**
Color contrast ratio for “Sign In” buttons should be >= 4.5:1.

**User Impact:**
Low vision users can not see complete information if color contrast ratio is not proper.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
![4 CC Sign in](https://user-images.githubusercontent.com/38523250/62203740-5fd2b480-b3a9-11e9-9d48-7ebf55ceeea2.png)

**Comment by Flanker32** [comment]
The color of button are affected by IntelliJ theme, which is designed by IntelliJ and out of the scope of Toolkits. Besides, IntelliJ provide three theme by default and one of them is High contrast, which is designed for low vision users by high color contrast ratio, here is the screen shots.

![image](https://user-images.githubusercontent.com/12445236/63919073-9535f500-ca70-11e9-9daf-650eaedf38f5.png)

![image](https://user-images.githubusercontent.com/12445236/63919302-0e354c80-ca71-11e9-86a5-27bcb5b1a79d.png)

**Comment by Flanker32** [comment]
- Importance of experience: Medium, low vision users may can not see complete information if color contrast ratio is not proper.
- Quality of experience: not impacted. IntelliJ provide alter theme for users including the High contrast one with high color contrast ratio.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@Accessibility56 Please have a review.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
Linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @soumyatippapuram please help us review it, thanks!

**Comment by Accessibility56** [comment]
Logged an External bug: https://youtrack.jetbrains.com/issue/IDEA-222112

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749780
